antora-playbook.yml

```diff
@@ -3,7 +3,10 @@ site:
   # features that depend on the site URL: https://docs.antora.org/antora/2.3/playbook/site-url/#when-should-the-site-url-be-set
   url: https://documentation.bonitasoft.com/
   start_page: bonita::index.adoc
-  robots: allow
+  robots: |
+    User-agent: *
+    Allow: /0/archives/release-notes*
```

Comment:
⚠️ this page doesn't exist and so generates a 404 error, so I don't understand why it is allowed 😸 
https://documentation.bonitasoft.com/0/archives/release-notes

Note that the page does not exist in the bonita component either: https://documentation.bonitasoft.com/bonita/0/archives/release-notes.html

If we have links leading to such urls, they probably come from a bad redirects configuration and we should fix them.